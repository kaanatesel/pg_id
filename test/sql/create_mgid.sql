CREATE TABLE test_mgid1 (a mgid);
INSERT INTO test_mgid1 VALUES ('507f191e810c19729de860ea');
SELECT a FROM test_mgid1;
INSERT INTO test_mgid1 VALUES ('507f191e810c19729d');
INSERT INTO test_mgid1 VALUES ('507f191e810c19729de960ea0');