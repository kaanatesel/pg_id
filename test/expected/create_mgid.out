CREATE TABLE test_mgid1 (a mgid);
INSERT INTO test_mgid1 VALUES ('507f191e810c19729de860ea');
SELECT a FROM test_mgid1;
            a             
--------------------------
 507f191e810c19729de860ea
(1 row)

INSERT INTO test_mgid1 VALUES ('507f191e810c19729d');
ERROR:  invalid input syntax for type mgid: "507f191e810c19729d"
LINE 1: INSERT INTO test_mgid1 VALUES ('507f191e810c19729d');
                                       ^
INSERT INTO test_mgid1 VALUES ('507f191e810c19729de960ea0');
ERROR:  invalid input syntax for type mgid: "507f191e810c19729de960ea0"
LINE 1: INSERT INTO test_mgid1 VALUES ('507f191e810c19729de960ea0');
                                       ^
